packages/components/src/components/toaster/toaster.tsx

```diff
@@ -48,8 +48,12 @@ export class ToasterService {
 		 * so we can't enqueue toasts.
 		 */
 		if (this.toastContainerElement && typeof this.toastContainerElement.enqueue === 'function') {
```

Comment:
Die Logik funktioniert bei meinen Tests leider nicht.

Beispiel:

```
const toaster = ToasterService.getInstance(document, {
		defaultVariant: 'card',
	});

void toaster.enqueue({
			description: 'Toasty',
			label: `Initial Toast`,
			type: 'warning',
			variant: 'msg',
		});
```

Erwartung: Toast der Variante msg wird gerendert.
Tatsächlich: Toast wird als card gerendert.

Hier am besten einmal alle Varianten durchtesten, ich habe dafür das Toast Basic Beispiel in der Sample App angepasst.